editoast/src/views/timetable.rs

```diff
@@ -449,21 +458,49 @@ impl Conflict {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
-struct PacedTrainOccurrenceId {
-    paced_train_id: i64,
-    index: u64,
+#[serde(untagged)]
+enum ConflictPacedTrainId {
+    Occurence {
+        paced_train_id: i64,
+        index: u64,
+    },
+    Exception {
+        paced_train_id: i64,
+        exception_key: String,
+        index: Option<u64>,
+    },
```

Comment:
I'm not sure that I understand why the `index` is optional? Is it because it depends if the exception is about a modified occurrence (it has an index), or if the exception is an added occurrence (in this case, it has not index)? If this is the explanation, then I would rather make a third variant in the `enum`.